File: 08.feetWet/09.expresso/01.hamcrest/app/src/main/java/info/juanmendez/hellohamcrest/Song.java
package info.juanmendez.hellohamcrest;

/**
 * Created by musta on 8/20/2016.
 */
public class Song {
    private final long id;
    private String title;
    private String video_url;
    private int year;

    public Song(long id, String title, String url, int year) {
        this.id = id;
        this.title = title;
        this.video_url = url;
        this.year = year;
    }

    public long getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getVideo_url() {
        return video_url;
    }

    public int getYear() {
        return year;
    }

}

File: 09.local_storage_db/08.object_box/app/src/main/java/info/juanmendez/objectbox/models/Song.kt
package info.juanmendez.objectbox.models

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

@Entity
data class Song(//@PrimaryKey
        @Id var id: Long = 0, //required to be a var, ObjectBox assigns id
        val title: String,
        val video_url: String,
        val year: Int)